ILTransform/ILRewriter.cs

```diff
@@ -237,7 +237,12 @@ private void RewriteFile(string source)
                 {
                     if (isILTest)
                     {
-                        string classLine = $".class public auto ansi Test_{Path.GetFileNameWithoutExtension(source)} extends [mscorlib] System.Object {{";
+                        string testClass = Path.GetFileNameWithoutExtension(source);
+                        if (!Char.IsLetter(testClass[0]))
```

Comment:
another "do you use VS" questions - I get this `Char` in a faded color in VS as the standard to use the C# special names (`char` here rather than `Char` which is resolving as `System.Char`)

Reply:
It seems I may be losing some of these suggestions from the IDE.